root/edit/details/historic/convert_release_to_multiple_artists.tt

```diff
@@ -9,7 +9,7 @@
         [% END %]
       </ul>
       [% ELSE;
-           l('(unknown)');
+          lp('(unknown)', 'release');
```

Comment:
I'm confused about the original indentation here.

Reply:
Looks like it was indented two spaces after the start of the `ELSE` (since the file uses two spaces indentation). Though the file as a whole isn't very consistent.